djangocms_navigation/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from treebeard.admin import TreeAdmin
+from treebeard.forms import movenodeform_factory
+
+from .models import Menu, MenuContent, MenuItem
+
+
+class MenuContentAdmin(admin.ModelAdmin):
+    exclude = ['modified_by', 'menu', ]
+
+    def save_model(self, request, obj, form, change):
+        if not obj.pk:
```

Comment:
After changing the one2one to an FK, this will need to be `if not obj.menu`.

Reply:
This still won't work for obj.menu. django will throw `RelatedObjectDoesNotExist` on check. I will keep obj.pk here.